libnavigation-base/src/main/java/com/mapbox/navigation/base/speed/model/SpeedLimit.kt

```diff
@@ -0,0 +1,7 @@
+package com.mapbox.navigation.base.speed.model
+
+data class SpeedLimit(
```

Comment:
Are we using a `data class` here (instead of a `Builder`) because we know that is not going to change eventually?

Reply:
Yes. `Builder` doesn't make sense here because it's not something that can have default values and a potential to have data that can be overridden. Once the `MapMatcherResult` prepares this `object` it's going to live unless another `routeProgress` event is emitted.